index.html

```diff
@@ -58,7 +58,7 @@
 
 <div class="wrapper">
     <div class="page-header clear-filter" filter-color="orange">
-      <div class="page-header-image" data-parallax="true" style="background-image:url('tiramisu3.jpg');">
+      <div class="page-header-image" data-parallax="true" style="background-image:url('backimage2.jpg');">
```

Comment:
do you that we can put all the picture in one folder in order to ease the visibility and to be sure that we are not going to use a wrong one ?